src/charm.py

```diff
@@ -51,6 +53,31 @@ def _on_config_changed(self, _: ops.ConfigChangedEvent) -> None:
         """Config changed handler."""
         self.reconcile()
 
+    def _on_install(self, _: ops.InstallEvent) -> None:
+        """Install event handler."""
+        if not self.state:
+            return
+        # This is the default user password
+        default_token = "Bearer wazuh:wazuh"  # nosec
+        try:
+            r = requests.put(
+                "https://localhost:55000security/users/2",
```

Comment:
Should it be `https://localhost:55000/security/users/2` instead of `https://localhost:55000security/users/2`?

Reply:
Oops. Good catch